### ISSUE TITLE
GPL-716 - As developers we need to ensure that all of the squix printers are setup and working

### ISSUE THREAD

**Issue by stevieing** [sanger/General-Backlog-Items#63]
**User story**
As developers we need to ensure that all of the squix printers are setup and working:

- [x] heron-bc1
- [x] heron bc2
- [x] heron-bc3
- [x] heron-bc4
- [x] heron-bc5
- [x] heron-bc6
- [x] heron-bc7

**Who are the primary contacts for this story**
Matt M (In Systems team)

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] The web interface of the printer is viewable and editable
- [ ] ftp is enabled with password
- [ ] http is enabled as simple with password
- [ ] ACL is enacted (hc6 emailed Service Desk on 22nd)
- [ ] printer is available via ftp from FCE
- [ ] Confirm what label size each printer uses (Lesley S confirming by Monday)

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**
Link to confluence document [here](https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/SQUIX+printing) outlining procedure for access and who to talk to

**Comment by stevieing** [comment]
Hi Steve,
 
I’ve added the details for each printer below;
 
heron-bc1 - 96 well plate
heron-bc2 – 384 well plate
heron-bc3- 96 well plate
heron-bc4– 384 well plate
heron-bc5- 96 well plate
heron-bc6– 384 well plate
heron-bc7- Tubes
 
Best wishes,
Lesley

**Comment by stevieing** [comment]
Current state of play:
- heron-bc2 and heron-bc7 to be installed in Ogilvie somewhere. Other four printers will be set up when lab is ready in November.

**Comment by harrietc52** [comment]
heron-bc1 - 96 well plate (done. Thank-you.)
heron-bc2 - 384 well plate (possibly next?)
heron-bc3 - 96 well plate
heron-bc4 - 384 well plate
heron-bc5 - 96 well plate
heron-bc6 - 384 well plate
heron-bc7 - Tubes (possibly next?)

**Comment by rl15** [comment]
From stand up on 4th, 3 or 7 printers done. On hold till ~6th hen new lab released and printers can be networked.
